ui/components/multichain/account-details/account-details-key.js

```diff
@@ -50,13 +56,22 @@ export const AccountDetailsKey = ({ accountName, onClose, privateKey }) => {
           variant={TextVariant.bodySm}
           style={{ wordBreak: 'break-word' }}
         >
-          {privateKey}
+          <span onClick={() => setShowDisableSelectWarn(true)}>
+            <LavaDomeReact unsafeOpenModeShadow={inTest} text={privateKey} />
+          </span>
         </Text>
         <ButtonIcon
-          onClick={() => handlePrivateKeyCopy(privateKey)}
+          onClick={() =>
+            setShowDisableSelectWarn(false) || handlePrivateKeyCopy(privateKey)
+          }
```

Comment:
non-blocking: To improve the accessibility here we could add an `ariaLabel={t('copyPrivateKey')}` which would need to be created in `messages.json`. ariaLabel should be a required prop for this component

Reply:
https://github.com/MetaMask/metamask-extension/pull/22765/commits/7ceba331e977edf5d30c0bfa6e97baa04fb2b373